CMakeLists.txt

```diff
@@ -71,8 +71,11 @@ target_link_libraries (heat
 
 
 add_executable (mesh_geoheat EXCLUDE_FROM_ALL src/mesh_geoheat_demo.cpp)
-target_link_libraries (mesh_geoheat 
+target_include_directories (mesh_geoheat PUBLIC ${SuiteSparse_INCLUDE_DIRS})
```

Comment:
Since mesh_geoheat is just an executable, it is not necessary to declare the suitesparse headers as part of its public interface.

Reply:
I'm not really good with cmake, I'll defer to your opinion!